src/extensions/default/InlineColorEditor/CSSProperties.json

```diff
@@ -0,0 +1,230 @@
+{
+    "align-content":               {"values": ["center", "flex-end", "flex-start", "space-around", "space-between", "stretch"]},
```

Comment:
Is it possible for us to reduce the size of this to only those properties that actually need the color picker?  It seems like a waste to have all of these here, when there are very few that actually need to display the color picker.